spyderlib/utils/introspection/jedi_plugin.py

```diff
@@ -54,10 +54,9 @@ def load_plugin(self):
         self._warmup_thread.start()
 
     def get_completions(self, info):
-        """Return a list of completion strings"""
+        """Return a list of (completion, type) tuples"""
         completions = self.get_jedi_object('completions', info)
-        debug_print(str(completions)[:100])
```

Comment:
Why did you remove this `debug_print` here? Just curious, I don't how important that is.